packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/e2e/quanticStandaloneSearchBox.e2e.ts

```diff
@@ -0,0 +1,95 @@
+import {testStandaloneSearchBox as test, expect} from './fixture';
+
+const mockSuggestions = async (page) => {
+  await page.route('**/rest/search/v2/querySuggest*', async (route: any) => {
+    const response = await route.fetch();
+    const json = await response.json();
+    json.completions = [
+      {
+        expression: 'test',
+        highlighted: '[test]',
+      },
+      {
+        expression: 'test 2',
+        highlighted: '[test][2]',
+      },
+    ];
+    json.responseId = 21322434;
+    await route.fulfill({response, json});
+  });
+};
```

Comment:
it would be nice to align with @erocheleau 's PR on this regard: https://github.com/coveo/ui-kit/pull/5425

https://github.com/coveo/ui-kit/blob/f31a37a2c490be6d62951e89375467cd8a36a7cb/packages/quantic/playwright/page-object/querySuggestObject.ts